ui/component/channelDiscussion/index.js

```diff
@@ -1,6 +1,7 @@
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router';
 import { DISABLE_COMMENTS_TAG } from 'constants/tags';
```

Comment:
- [ ] On mobile, create like 14 layers of nested comments. Copy the link to the 14th comment, then paste into a new tab.  Tap "view all comments".  Try to expand the same thread again, and the thread mode fails:

<img width="482" alt="image" src="https://user-images.githubusercontent.com/64950861/167884672-9149e6b4-55f7-45b8-9efe-484fa1c6aedf.png">

Reply:
Pasting https://salt.odysee.com/@dev-test:8/tn:6?lc=f078dde2b6a5c2e5338c0bc51714cb9f77d0959fe6ab2d1dde8dab339974301e directly also fails 

![Uploading image.png…]()